vault/resource_kv_secret_v2_test.go

```diff
@@ -218,6 +212,8 @@ func TestAccKVSecretV2_DisableRead(t *testing.T) {
 
 // Fadia u have added this
 func TestAccKVSecretV2_UpdateOutsideTerraform(t *testing.T) {
+	// TODO skipping in CI for now. Determine if this is still a valid test case
```

Comment:
With the breaking change, this test-case may no longer be valid, as TFVP can not track updates outside of Terraform.

Reply:
Removed after determining this is no longer a valid test case, as going forward TFVP will not track drifts in secret data itself